doc/source/user_guide/solution.rst

```diff
@@ -28,15 +37,33 @@ The ``solve.set/gradient_scheme`` TUI sets the gradient options.
     
 Selecting solution controls 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The ``solve.set.p_v_controls`` TUI sets pressure-velocity controls.
+The following example shows a comparison between the TUI command and the
+python code for selecting the pressure velocity controls.
+
+TUI command
+
+.. code:: scheme
+
+    /solve/set/p-v-controls 0.3 0.4
```

Comment:
```suggestion
    ! Set momentum and pressure under-relaxation factors
    /solve/set/p-v-controls 0.3 0.4 
```
An exclamation point is a comment char right?

Reply:
@dnwillia-work Semicolon char.